### ISSUE TITLE
Crash on "Popups" tab.

### ISSUE THREAD

**Issue by Vethavear** [eigengrau-ch/strapi-plugin-cookie-manager#21]
After clicking on Popups tab in admin there is following error: 
```
TypeError
destroy is not a function
Call Stack
 safelyCallDestroy
  webpack-internal:///10344:22927:5
 commitHookEffectListUnmount
  webpack-internal:///10344:23095:11
 commitPassiveUnmountInsideDeletedTreeOnFiber
  webpack-internal:///10344:25093:11
 commitPassiveUnmountEffectsInsideOfDeletedTree_begin
  webpack-internal:///10344:25043:5
 commitPassiveUnmountEffects_begin
  webpack-internal:///10344:24951:11
 commitPassiveUnmountEffects
  webpack-internal:///10344:24936:3
 flushPassiveEffectsImpl
  webpack-internal:///10344:27033:3
 flushPassiveEffects
  webpack-internal:///10344:26979:14
 commitRootImpl
  webpack-internal:///10344:26930:5
 commitRoot
  webpack-internal:///10344:26677:5
```

From developer console I see that error is coming from CookieTab component.

Strapi version: 4.11.0
Plugin version: 1.1.2

**Comment by eigengrau-ch** [comment]
@Vethavear Thanks for reporting! I will look into it as soon as possible.

**Comment by eigengrau-ch** [comment]
Hi everyone, the issue has been solved by cef52eef in version 1.2.2. Thanks for reporting the issue
